openglyph/include/openglyph/steam/steam_paths.hpp

```diff
@@ -0,0 +1,30 @@
+#include <cstdint>
+#include <filesystem>
+#include <vector>
+namespace openglyph::steam {
```

Comment:
I don't think `steam` is important enough in OpenGlyph to warrant it's own namespace. It's just this file, pretty much.
Maybe `utility` for now?

Reply:
This comment hasn't been addressed. It's still in the `steam` include path, and in the `steam` namespace.